src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/regelmodell/resultat/Beregningsgrunnlag.java

```diff
@@ -33,7 +33,11 @@
     private boolean beregningForSykepenger = false; //Alltid false i FPSAK
     private boolean hattMilitærIOpptjeningsperioden = false;
     private boolean besteberegnet = false;
-    private int antallGrunnbeløpMilitærHarKravPå = 3; //TODO : Denne burde mappes for SVP (og eventuelt FP) fra f. eks konfig-verdi
+    private int antallGrunnbeløpMilitærHarKravPå = 3;
+    private BigDecimal øvreGrenseverdiAntallGBg;
```

Comment:
No blei det mykje fram og tilbake her, men kanskje vi burde standarisere måten vi skriver "dette beløpet er eit tall som representerer antall G". At man då kanskje endret dette til antallGØvreGrenseverdi for å matche antallGMinstekravVilkår

Reply:
Enig i det, endrer militærvariabelen og da

Reply:
Det vil vel knekke kalkulus men er vel bare jeg som skal bumpe uansett